code/modules/reagents/newchem/other.dm

```diff
@@ -120,7 +120,7 @@ datum/reagent/colorful_reagent
 	mix_message = "The substance flashes multiple colors and emits the smell of a pocket protector."
 
 datum/reagent/colorful_reagent/reaction_mob(var/mob/living/simple_animal/M, var/method=TOUCH, var/volume)
-    if(M)
+    if(M && istype(M))
         M.color = pick(random_color_list)
```

Comment:
Odd. As far as I can tell, the code you added doesn't change the HTML or the way languages are set; it simply outputs a specific message when certain default languages are set. *Before* the default language is updated. Actually, it will list the wrong language if your default is set to the Unathi or Tajaran languages and you change it to another, which may have been why it appeared to work - if you used Set Default Language to pick one of them, opened the Check Known Languages dialog, and clicked their Reset link, it would say you set that language, but actually unset it.